src/sdk/Exports.ts

```diff
@@ -50,3 +50,17 @@ export { ActivityReceivedEventArgs } from "./ActivityReceivedEventArgs";
 export { ServicePropertyChannel } from "./ServicePropertyChannel";
 export { ProfanityOption } from "./ProfanityOption";
 export { BaseAudioPlayer } from "./Audio/BaseAudioPlayer";
+export { Conversation,
+    ConversationExpirationEventArgs,
+    ConversationImpl,
```

Comment:
>ConversationImpl [](start = 4, length = 16)

Why are we exporting a Impl class on the public SDK interface?

Reply:
Updated code to remove this export as it's not required here.